packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,540 @@
+                aria-label={formatMessage({
+                    id: 'shipping_multi_address.error.label',
+                    defaultMessage: 'Error loading products'
+                })}
+            >
+                {formatMessage({
+                    id: 'shipping_multi_address.error.message',
+                    defaultMessage: 'Error loading products. Please try again.'
+                })}
+            </Box>
+        )
+    }
+
+    const onAddNewAddress = () => {
+        // TODO: Implement add new address logic/modal
+        console.log('Add New Address clicked!')
+    }
+
+    return (
+        <Box
+            as="main"
+            role="main"
+            aria-label={formatMessage({
+                id: 'shipping_multi_address.main.label',
+                defaultMessage: 'Multi-shipping address selection'
+            })}
+        >
+            <VStack spacing={0}>
+                <Box
+                    border="1px solid"
+                    borderColor="gray.200"
+                    borderRadius="md"
+                    bg="white"
+                    p={2}
+                    w="100%"
+                    role="region"
+                    aria-label={formatMessage({
+                        id: 'shipping_multi_address.products.region',
+                        defaultMessage: 'Products and delivery addresses'
+                    })}
+                >
+                    <VStack spacing={2}>
+                        {basket.productItems.map((item) => {
+                            // Merge product details into item
+                            const productDetail = productsMap?.[item.productId] || {}
+                            const variant = {...item, ...productDetail}
+                            // Use findImageGroupBy to get the image
+                            const image = findImageGroupBy(productDetail.imageGroups, {
+                                viewType: 'small',
+                                selectedVariationAttributes: variant.variationValues
+                            })?.images?.[0]
+                            const imageUrl = image?.disBaseLink || image?.link || ''
+                            const addressKey = item.itemId
+                            const selectedAddressId =
+                                selectedAddresses[addressKey] || addresses[0]?.addressId
+
+                            return (
+                                <Box
+                                    key={addressKey}
+                                    border="1px solid"
+                                    borderColor="gray.200"
+                                    borderRadius="md"
+                                    p={4}
+                                    bg="white"
+                                    w="100%"
+                                    data-testid="multi-shipping-card"
+                                    overflow="visible"
+                                    role="article"
```

Comment:
How does this fit into the role=article? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/article_role